java/trino/src/main/java/sleeper/trino/SleeperMetadata.java

```diff
@@ -216,10 +217,12 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
             List<ColumnHandle> keyColumnHandles = sleeperTableHandle.getColumnHandlesInCategoryInOrder(SleeperColumnHandle.SleeperColumnCategory.ROWKEY).stream()
                     .map(ColumnHandle.class::cast)
                     .collect(ImmutableList.toImmutableList());
-            // Generate as list of the lower-bounds of eadh partition and use these to create a SleeperPartitioningHandle object.
+
+            Schema tableSchema = this.sleeperConnectionAsTrino.getSchema(sleeperTableHandle.getSchemaTableName());
```

Comment:
The use of this here is unnecessary, see https://github.com/gchq/sleeper/blob/develop/docs/development/conventions.md#use-of-this.